site/app/controllers/grading/ElectronicGraderController.php

```diff
@@ -2373,4 +2373,23 @@ private function getStats(Gradeable $gradeable, User $grader, bool $full_stats,
             $total_total += $value * $num_components;
         }
     }
+    
+    /**
+     * @Route("/courses/{_semester}/{_course}/gradeable/{gradeable_id}/feedback/set", methods={"POST"})
+     */
+    public function setPeerFeedback($gradeable_id) {
```

Comment:
If this is an ajax endpoint, rename this function `ajaxSerPeerFeedback` and add some access control.

Reply:
Should there be access control? Everone should be able to give peer feedback, or am I misunderstanding access control?

Reply:
I have changed to ajaxSetPeerFeedback